lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Action.js

```diff
@@ -246,18 +243,63 @@ qx.Class.define("callbackery.ui.plugin.Action", {
         _addUploadButton: function(cfg,btCfg,getFormData){
-            var form = new uploadwidget.UploadForm('uploadFrm','upload');
-            form.setParameter('name',cfg.name);
-            form.setParameter('key',btCfg.key);
-            form.setLayout(new qx.ui.layout.HBox());
-            this.add(form);
             var file;
             if (btCfg.btnClass == 'toolbar') {
-                file = new uploadwidget.UploadToolbarButton('file', this.xtr(btCfg.label));
+                file = new callbackery.ui.form.UploadToolbarButton(this.xtr(btCfg.label));
             }
             else {
-                file = new uploadwidget.UploadButton('file', this.xtr(btCfg.label));
+                file = new callbackery.ui.form.UploadButton(this.xtr(btCfg.label));
             }
+
+            let that = this;
+            file.addListener('changeFileName', function(e){
+                let formData = new FormData();
+                formData.append("file", e.getData());
+                formData.append("name", "onboardingBulkUpload");
+                formData.append("key", "upload_bulk_invite");
+
+                callbackery.data.Server.getInstance().callAsyncSmart(function(cookie){
+                    formData.append('xsc',cookie);
+                    let req = new qx.io.request.Xhr("/upload", "POST");
+                    req.addListener("success", function(e) {
+                        let req = e.getTarget();
+                        file.clear(); // So we can upload the same file multiple times in a row.
+                        let response = {};
+                        try {
+                            response = qx.lang.Json.parse(req.getResponse());
+                        }
+                        catch(e){
+                            response = {
+                                exception: {
+                                    message: e.message,
+                                    code: 99999
+                                }
+                            };
+                        }
+                        if (response.exception){
+                            callbackery.ui.MsgBox.getInstance().error(
+                                that.tr("Upload Exception"),
+                                that.xtr(response.exception.message) + " ("+ response.exception.code +")"
+                            );
+                            return;
+                        }
+                        if (response){
+                            that.fireDataEvent('actionResponse',response || {});
+                        }
+                    }, this);
+                    req.addListener("error", function(e) {
+                        callbackery.ui.MsgBox.getInstance().error(
+                            that.tr("Network error"),
+                            that.tr("Upload of file failed")
+                        );
+                        return;
+                    }, this);
+
+                    req.send(formData);
```

Comment:
das ist falsch .. send nimmt kein argument
```suggestion
                    req.setRequestData(formData);
                    req.send();
```